packages/sdk/src/Products.ts

```diff
@@ -7,67 +9,73 @@ export class Products {
+	public update(product: T.ProductsUpdate) {
+		return this.client.put<T.ProductsUpdateResponse>(
+			`/products/${encode(product.id || product.source_id)}`,
+			omit(product, ['id']),
+		)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-products-in-bulk
 	 */
-	public bulkUpdate(products: $FixMe) {
-		return this.client.post('/products/bulk', products)
+	public bulkUpdate(products: T.ProductsBulkUpdate) {
+		return this.client.post<T.ProductsBulkUpdateResponse>('/products/bulk', products)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-product
 	 */
-	public delete(productId: $FixMe, params?: $FixMe) {
+	public delete(productId: string, params?: T.ProductsDeleteParams) {
 		return this.client.delete(`/products/${encode(productId)}`, { qs: params })
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-products
 	 */
-	public list(params?: $FixMe) {
-		return this.client.get('/products', params)
+	public list(params?: T.ProductsListParams) {
+		return this.client.get<T.ProductsListResponse>('/products', params)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-sku
 	 */
-	public createSku(productId: string, sku: $FixMe) {
-		return this.client.post(`/products/${encode(productId)}/skus`, sku)
+	public createSku(productId: string, sku: T.ProductsCreateSku) {
+		return this.client.post<T.ProductsCreateSkuResponse>(`/products/${encode(productId)}/skus`, sku)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#get-sku
 	 */
 	public getSku(productId: string, skuId: string) {
-		return this.client.get(`/products/${encode(productId)}/skus/${encode(skuId)}`)
+		return this.client.get<T.ProductsGetSkuResponse>(`/products/${encode(productId)}/skus/${encode(skuId)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-sku
 	 */
-	public updateSku(productId: string, sku: $FixMe) {
-		return this.client.put(`/products/${encode(productId)}/skus/${encode(sku.id || sku.source_id)}`, omit(sku, ['id']))
+	public updateSku(productId: string, sku: T.ProductsUpdateSku) {
+		return this.client.put<T.ProductsUpdateSkuResponse>(
+			`/products/${encode(productId)}/skus/${encode(sku.id || sku.source_id)}`,
+			omit(sku, ['id']),
+		)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-sku
 	 */
-	public deleteSku(productId: string, skuId: string, params?: $FixMe) {
+	public deleteSku(productId: string, skuId: string, params?: T.ProductsDeleteSkuParams) {
 		return this.client.delete(`/products/${encode(productId)}/skus/${encode(skuId)}`, { qs: params })
```

Comment:
This is an error, as second parameter it should just be `params` and not `{ qs: params}`.